provider/gandi/gandi.go

```diff
@@ -113,16 +151,24 @@ func (p *GandiProvider) Records(ctx context.Context) ([]*endpoint.Endpoint, erro
 		}
 
 		for _, r := range records {
+			if isGandiAlias(r, p.PreferALIAS) {
+				// Convert back ALIAS to CNAME
+				r.RrsetType = endpoint.RecordTypeCNAME
```

Comment:
Shouldn't this set a provider-specific property to indicate the underlying record is an alias?

Reply:
This is already manipulating a provider-specific object - Gandi's `livedns.DomainRecord`

Do you think this should be set before in the `endpoint.ProviderSpecificProperty` prop and reused later?